src/Component/JsonSchema/Guesser/Validator/String_/MinLengthValidator.php

```diff
@@ -29,5 +30,8 @@ public function guess($object, string $name, $guess): void
             'min' => $object->getMinLength(),
             'minMessage' => 'This value is too short. It should have {{ limit }} characters or more.',
         ]));
+        if ($object->getMinLength() > 0) {
```

Comment:
we could probably even skip the whole min length validation if the value is 0. not sure why it is generated in that case. i can also try that if you agree, but somewhat worried it might lead to empty constraint lists...

Reply:
I think it's fine that way :+1: